core/server/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1891,7 +1891,7 @@ private long getWorkerStoringFile(TachyonURI path) throws FileDoesNotExistExcept
   public synchronized FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFiles)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException {
     for (long fileId : persistedFiles) {
-      setState(getPath(fileId), SetAttributeOptions.defaults().setPersisted(true));
+      setState(getPath(fileId), new SetAttributeOptions.Builder().setPersisted(true).build());
```

Comment:
do we give up the `defaults()` static method and go back to the `Builder` model?